tests/Model/Domain/TrackingResponseTest.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * Copyright (C) 2013 Mailgun
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license. See the LICENSE file for details.
+ */
+
+namespace Mailgun\Tests\Model\Domain;
+
+use Mailgun\Model\Domain\ClickTracking;
+use Mailgun\Model\Domain\OpenTracking;
+use Mailgun\Model\Domain\TrackingResponse;
+use Mailgun\Model\Domain\UnsubscribeTracking;
+use Mailgun\Tests\Model\BaseModelTest;
+
+class TrackingResponseTest extends BaseModelTest
+{
+    public function testCreate()
+    {
+        $json =
+            <<<'JSON'
+{
+    "tracking": {
+        "click": {
+            "active": "htmlonly"
+        },
+        "open": {
+            "active": "no"
```

Comment:
According to the example at the end of the page of the API docs, this one seems to be a boolean value

```suggestion
            "active": false
```

https://documentation.mailgun.com/en/latest/api-domains.html#example

![image](https://user-images.githubusercontent.com/7809429/95923642-d93ba480-0dad-11eb-8699-dabb86e7d026.png)